src/bin/io/prompt.rs

```diff
@@ -213,20 +217,23 @@ fn check_valid_denom(value: f64, balance: Dusk) -> Result<(), String> {
 }
 
 /// Request amount of tokens
-pub(crate) fn request_token_amt(action: &str, balance: Dusk) -> Dusk {
+pub(crate) fn request_token_amt(
+    action: &str,
+    balance: Dusk,
+) -> anyhow::Result<Dusk> {
     let question = requestty::Question::float("amt")
         .message(format!("Introduce the amount of DUSK to {}:", action))
         .default(MIN_CONVERTIBLE.into())
         .validate_on_key(|f, _| check_valid_denom(f, balance).is_ok())
         .validate(|f, _| check_valid_denom(f, balance))
         .build();
 
-    let a = requestty::prompt_one(question).expect("token amount");
-    a.as_float().unwrap().into()
+    let a = requestty::prompt_one(question)?;
+    Ok(a.as_float().unwrap().into())
```

Comment:
I didn't find a way to remove that unwrap, due to limitation in `requestty` API.
Thinking about it, it should be an `expect` to be honest, because that should be guaranteed by how the `question` is built
In this specific case it's:
`requestty::Question::float(` so I can expect that `.as_float()` is valued

Reply:
So I would change it to 
```rust
Ok(a.as_float().expect("answer to be a float").into())
```

Reply:
Another way, super verbose imo but at least with no expect/unwrap at all, can be
```rust
    match requestty::prompt_one(question)? {
        requestty::Answer::Float(val) => Ok(val.into()),
        _ => Err(anyhow::anyhow!("Invalid answer")),
    }
 ```

Reply:
You could use `try_into_float()`, however you would most likely still need to return a custom error, so it would be still verbose:
```rust
        Ok(a.try_into_float()
            .map_err(|_| anyhow::anyhow!("Float conversion error"))?
            .into());
```

Unfortunately I don't think `requestty` implements `std::error::Error`.

Let's keep the `expect` for the time being, once we're removing the library we'll search for something better.